readme.md

```diff
@@ -12,7 +12,7 @@ Following steps have been tested on Ubuntu 20.04 LTS. When using a virtual machi
 1. take `<xxx>.rpi-sdimg` from `build/tmp/deploy/images/raspberrypi4/` and write it to an SD card
     1. `sudo fdisk -l | grep /dev/sd` to determine device
     1. `sudo dd if=<xxx>.rpi-sdimg of=/dev/sd<x> bs=4M` to write image on device
-1. startup Raspberry Pi and connect via UART or SSH
+1. startup Raspberry Pi and connect via UART or SSH (first startup takes time as filesystem is expanded to available memory on SD card)
```

Comment:
How much time are we talking about? a couple of seconds or a couple of minutes?

Reply:
Kommt auf die Grösse der SD-Karte draufan. Meine ist 16GB und es dauerte knapp 4 Minuten. Soll ich es genauer angeben?

Reply:
Ja bitte schreib doch dort noch die Zahlen hin. Weil bei 4 Minuten könnte man sonst schon langsam denken dass etwas nicht richtig funktioniert hat.